assets/js/api/_load.js

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';
```

Comment:
This is also imported in `components.js` which is bundled with this file in the gulp build task. Could this result in some duplication in `bundle.js` even with uglify running?

Reply:
Not related to changes in the context of this PR, create an issue?